src/loops/while_loops/hard/q8/CounterChar.java

```diff
@@ -0,0 +1,27 @@
+package loops.while_loops.hard.q8;
+
+/*
+Write a program that asks the user for a sentence and then counts the number of characters in the sentence using a while loop.
+(Hint: Space/' ' is not considered a character)
```

Comment:
I would not put this as a hint, just say something like `Assume space is not a character.`